easybuild/easyblocks/l/llvm.py

```diff
@@ -869,7 +874,7 @@ def configure_step(self):
                 self.log.warning("`LLVM_HOST_TRIPLE` not found in the output of the configure step")
 
         if not self.cfg['bootstrap']:
-            if build_option('rpath') and self._cmakeopts['LLVM_ENABLE_RUNTIMES'] != '""':
+            if build_option('rpath') and self._cmakeopts['LLVM_ENABLE_RUNTIMES'] not in ('""', "''", ''):
```

Comment:
Since the expected value is tied to the result of `list_to_cmake_arg` we might want to do

```suggestion
            if build_option('rpath') and self._cmakeopts['LLVM_ENABLE_RUNTIMES'] != self.list_to_cmake_arg([]):
```

Alternatively we could just check `self.final_runtimes` here

```suggestion
            if build_option('rpath') and self.final_runtimes:
```

Reply:
If `bootstrap` is False it is always gonna be the final one, still no easy safeguard if the logics get changed in the future

Reply:
Checking `_cmakeopts` would guard against behavior changes on `self.final_runtimes` but i really do not think we should ever touch that as it is already fairly clear.

So yes i think the 2nd option is cleaner